capabilities/core-and-supporting.md

```diff
@@ -0,0 +1,27 @@
+---
+displayName: Core and Supporting
+epicsLabels:
+  - area/core-and-supporting
+  - area/community
+---
+
+## Scope
+
+The Core and Supporting capability provides functionality for delivery of the content and its visual representation. For us, the content is not only regular documentation but also specifications and contextual help.
+Due to the nature of the content and the number of different areas, it sits in, the Core and Supporting capability provides also many generic tools that do not only support content but also other aspects of the product.
+
+In other words, if some content must be displayed in a given UI, the capability cares also about the rest of the UI of a given business functionality and its backend.
+
+## Vision
+
+- Content is written once and reusable in different contexts: documentation portal, inline help or UI applications. This is possible because of:
+  - Kubernetes native way of content delivery and support for distributed content sourcing.
```

Comment:
I miss explicit mention of modularity and extensibility in the area of content sourcing and the backend that supports it. 
I mean the statement that content related to different modules (documentation and data) can be easily added and served efficiently.

Reply:
added although modularization for me is not a per capability vision, this is our new manifest element actually, that all new components that we add need to be modular and enabled for explicit enablement/disablement